app/core/src/screens/homepage/Homepage.js

```diff
@@ -2,9 +2,9 @@
 
 import * as React from 'react';
 import { View } from 'react-native';
-import { Button, Layout, Text } from '@kiwicom/react-native-app-shared';
+import { Button, Layout } from '@kiwicom/react-native-app-shared';
 import { type NavigationType } from '@kiwicom/react-native-app-navigation';
-import Translation from '@kiwicom/react-native-app-translations';
+import { DummyTranslation } from '@kiwicom/react-native-app-translations';
```

Comment:
Now it's time to think about a better name for "DummyTranslation"... 😄

Reply:
Agreed!

Reply:
Ideas?

Reply:
I will sleep on it 🛌